recipes/sensormapgeo/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "sensormapgeo" %}
+{% set version = "0.4.5" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/sensormapgeo-{{ version }}.tar.gz
+  sha256: 4f435416cb98602882e0a122a5499b8ffb027a36b6eae5529e493ae235a5cfcd
+
+build:
+  number: 0
+  skip: true  # [py<36]
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  build:
+    # force the use of llvm to fix https://gitext.gfz-potsdam.de/EnMAP/sensormapgeo/-/issues/6
```

Comment:
Ah, I didn't realize that you actually wanted LLVM openmp. Then what you had was correct.